.idea/misc.xml

```diff
@@ -1,6 +1,5 @@
-<?xml version="1.0" encoding="UTF-8"?>
 <project version="4">
-  <component name="ProjectRootManager" version="2" languageLevel="JDK_11" default="true" project-jdk-name="JDK" project-jdk-type="JavaSDK">
+  <component name="ProjectRootManager" version="2" languageLevel="JDK_15" default="true" project-jdk-name="15" project-jdk-type="JavaSDK">
```

Comment:
What does this change do? I'm leery of upgrading the JDK in case it breaks compatibility for the old OS versions the app supports. I was able to compile and run it without this change.

Reply:
That's just a change for my IDE. For.some reason I can only compile when it's set to JDK 15
In the next pr I will review in more detail so the unnecessary files won't be pushed